package.json

```diff
@@ -275,13 +268,13 @@
     "react-native-text-size": "rainbow-me/react-native-text-size#15b21c9f88c6df0d1b5e0f2ba792fe59b5dc255a",
     "react-native-tooltip": "rainbow-me/react-native-tooltip#e0e88d212b5b7f350e5eabba87f588a32e0f2590",
     "react-native-tooltips": "rainbow-me/react-native-tooltips#fdafbc7ba33ee231229f5d3f58b29d0d1c55ddfa",
-    "react-native-turbo-haptics": "1.0.4",
+    "react-native-turbo-haptics": "1.0.5",
     "react-native-udp": "2.7.0",
     "react-native-url-polyfill": "2.0.0",
     "react-native-version-number": "0.3.6",
     "react-native-video": "6.3.0",
     "react-native-view-shot": "4.0.2",
-    "react-native-vision-camera": "4.4.2",
+    "react-native-vision-camera": "4.6.4",
```

Comment:
could I get a confirmation of PoW for the camera upgrade on ios and android still working as expected

Reply:
<img width="379" alt="image" src="https://github.com/user-attachments/assets/2a5119fb-e918-496b-88a1-a4ec16eb9ea7" />

![IMG_3811](https://github.com/user-attachments/assets/b8094de8-219a-4a7c-bbf3-37397994f1f4)

I don't have a real Android device on hands currently, but was able to test on emulator. I also didn't test scanning an actual QR code as I am not sure exactly how that works. LMK if that's ok.